packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -148,5 +155,33 @@ cbp-list {
         padding-inline-end: var(--cbp-space-3x);
       }     
     }
+  }
+  
+  //Description List
+
+  dl{
+    padding-inline-start: var(--cbp-space-12x);
+
+    dt{
+      color: var(--cbp-desc-list-color);
+      font-size: var(--cbp-font-size-heading-sm);
+      font-weight: var(--cbp-font-weight-medium);
+      line-height: var(--cbp-line-height-sm);
+      padding-block-end: var(--cbp-space-2x);
+
+      cbp-icon{
+        margin-inline-start: calc(var(--cbp-space-9x) * -1);
+        padding-inline-end: var(--cbp-space-3x);
+      }
+    }
+
+    dd{
+      color: var(--cbp-desc-list-description-color);
+      font-size: var(--cbp-font-size-body);
```

Comment:
if you don't need to set these explicitly to override inherited/wrong values, remove them (font-size and weight).